tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Program.cs

```diff
@@ -119,6 +119,10 @@ private PipelineConvention GetPipelineConvention(string convention, PipelineGene
                     var upLogger = serviceProvider.GetService<ILogger<UnifiedPipelineConvention>>();
                     return new UnifiedPipelineConvention(upLogger, context);
 
+                case "upweekly":
```

Comment:
nit: for consistency it might be worth updating "weekly" below to be "testsweekly" just so it is clear.